starfish/util/plot.py

```diff
@@ -187,7 +187,24 @@ def diagnose_registration(
     **kwargs,
 ):
     """
-    # TODO docme
+    Overlays 2-d images extracted from ImageStack for the purpose of visualizing alignment of
+    images from different rounds or channels selected with the `sel` parameter. Up to six images
+    can be selected and shown in different colors. The same `Axes.X` and `Axes.Y` indices should
+    be used for every Selector.
+
+    Parameters
+    ----------
+    imagestack : ImageStack
+        imagestack from which to extract 2-d images for plotting
+    sel : Optional[Mapping[Axes, Union[int, tuple]]]
```

Comment:
why is this *sels but kwargs is not **kwargs?

Reply:
I don't understand the question. :(

Reply:
I'm trying to understand the conventions for docstring. You think "sel" should be "*sels" and I understand the asterisk is because it's a varargs. I am wondering why you didn't also comment that "kwargs" should be "**kwargs".

Reply:
Ohh, I guess `sels` is fine too.  I'm not very sure how to type a vararg parameter set though.

If you want it to be strictly typed, you could change sels to be an explicit sequence of `Mapping[Axes, Union[int, tuple]]`.